### PR TITLE
[chore] Run porto after go generate

### PR DESCRIPTION
**Pull request by pellared**
The same is done in the "main" (non-contrib) repository: https://github.com/open-telemetry/opentelemetry-go/blob/main/Makefile#L86

This can help addressing https://github.com/open-telemetry/opentelemetry-go-contrib/pull/3817#discussion_r1195242653

### CODE REVIEW

File: instrgen/driver/testdata/interface/serializer/interface.go [hunk 1]
```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 //nolint:all // Linter is executed at the same time as tests which leads to race conditions and failures.
-package serializer
+package serializer	// import "go.opentelemetry.io/contrib/instrgen/testdata/interface/serializer"
```
Comment (pellared) [inline instrgen/driver/testdata/interface/serializer/interface.go:16]: @pdelewski I do not understand why `instrgen_test.go` keeps replacing "space" to a "tab". Is it not a bug?

Comment (pellared) [reply instrgen/driver/testdata/interface/serializer/interface.go]: > Regarding tests, we can improve that, however it does not seem must have for now.

The problem is that the test execution modifies the `instrgen/driver/testdata/interface/app/impl.go` and `instrgen/driver/testdata/interface/serializer/interface.go` files that you can accidently commit.

File: Makefile [hunk 2]
```diff
@@ -68,18 +68,21 @@ $(TOOLS)/dbotconf: PACKAGE=go.opentelemetry.io/build-tools/dbotconf
...
 .PHONY: generate build
 
 generate: $(OTEL_GO_MOD_DIRS:%=generate/%)
 generate/%: DIR=$*
-generate/%: | $(STRINGER)
+generate/%: | $(STRINGER) $(PORTO)
 	@echo "$(GO) generate $(DIR)/..." \
 		&& cd $(DIR) \
-		&& PATH="$(TOOLS):$${PATH}" $(GO) generate ./...
+		&& PATH="$(TOOLS):$${PATH}" $(GO) generate ./... && $(PORTO) -w .
```
Comment (pellared) [inline Makefile:85]: This approach adds the vanity imports even to `testdata`. See https://github.com/open-telemetry/opentelemetry-go-contrib/pull/3820#discussion_r1196279532

Maybe we could simply do something like following instead?


```make
.PHONY: generate go-generate build

generate: go-generate vanity-import-fix

go-generate: $(OTEL_GO_MOD_DIRS:%=generate/%)
go-generate/%: DIR=$*
go-generate/%: | $(STRINGER) $(PORTO)
	@echo "$(GO) generate $(DIR)/..." \
		&& cd $(DIR) \
		&& PATH="$(TOOLS):$${PATH}" $(GO) generate ./...
```make

**Review by pdelewski — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (MrAlias) [reply Makefile:85]: I'll wait for this ^ change before reviewing.

Comment (pellared) [reply Makefile]: @MrAlias Done: https://github.com/open-telemetry/opentelemetry-go-contrib/pull/3820/commits/80bd9cdf06ff98567dca692b06fafc29e37704db

File: instrgen/driver/testdata/interface/serializer/interface.go (hunk 1 above)
Comment (pellared) [reply instrgen/driver/testdata/interface/serializer/interface.go]: I reverted he changes as they are not longer needed but I suggest at looking at this problem anyway 😉 https://github.com/open-telemetry/opentelemetry-go-contrib/pull/3820/commits/42181219d44cdb80e8fd1fcb2a2cbe99f5a2e829

**Review by MrAlias — APPROVED** [review]

**Pull request merged**